packages/gatsby-transformer-remark/src/on-node-create.js

```diff
@@ -38,6 +38,7 @@ module.exports = async function onCreateNode(
     internal: {
       content,
       type: `MarkdownRemark`,
+      rawMarkdownBody: data.content,
```

Comment:
This shouldn't be in `internal`, it should be property of `markdownNode` (similar to `excerpt`)